lib/galaxy/model/__init__.py

```diff
@@ -5099,6 +5099,22 @@ def create_social_auth(cls, user, uid, provider):
         return instance
 
 
+class CustosAuthnzToken(RepresentById):
+
+    def __init__(self, user, external_user_id, provider, access_token, id_token, refresh_token, expiration_time, refresh_expiration_time, raw_token):
+        self.id = None
+        # 'user' is a backref to model.User
+        self.user = user
+        self.external_user_id = external_user_id
+        self.provider = provider
+        self.access_token = access_token
+        self.id_token = id_token
+        self.refresh_token = refresh_token
+        self.expiration_time = expiration_time
+        self.refresh_expiration_time = refresh_expiration_time
+        self.raw_token = raw_token
```

Comment:
I guess now that you're storing `access_token`, `id_token`, `refresh_token`, `expiration_time`, and etc., which are all part of the `raw_token`, you would not need to persist the `raw_token` it self.